src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,61 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    static final int RADIUS_DEFAULT = 10;
+    static final String COLOR_DEFAULT = "WHITE";
+    protected Random random = new Random();
+    protected Circle circle = new Circle(random.nextInt(),
+            new ColorSupplier().getRandomColor());
+    protected IsoscelesTrapezoid isoscelesTrapezoid
+            = new IsoscelesTrapezoid(random.nextInt(),
+            random.nextInt(), random.nextInt());
+    protected Rectangle rectangle = new Rectangle(random.nextInt(),
+            random.nextInt());
+    protected RightTriangle rightTriangle = new RightTriangle(random.nextInt(),
+            random.nextInt());
+    protected Square square = new Square(random.nextInt());
```

Comment:
move this code to methods responsible for it. Also, methods should be private and placed after public methods